numpy/random/mtrand/mtrand.pyx

```diff
@@ -3837,20 +3837,19 @@ cdef class RandomState:
         -------
         out : ndarray or scalar
             Drawn samples from the parameterized negative binomial distribution,
-            where each sample is equal to N, the number of trials it took to
-            achieve n - 1 successes, N - (n - 1) failures, and a success on the,
-            (N + n)th trial.
+            where each sample is equal to N, the number of successes that
+            occurred before n failures, and a failure on the (N + n)th trial.
```

Comment:
I guess the ", and a failure on the (N + n)th trial" must follow logically.  If you track n failures, then you only track until you have them all.

Reply:
Fair enough. How about "for a total of (N + n) trials"?